SConstruct

```diff
@@ -1697,17 +1697,17 @@ ruamel_min_version = parse_version('0.15.34')
 # Minimum pytest version assumed based on Ubuntu 20.04
 pytest_min_version = parse_version("4.6.9")
 
+# Pytest is required only to test the Python module
+check_for_pytest = any(
+    target.startswith("test-python") for target in COMMAND_LINE_TARGETS
```

Comment:
the command line target `test` should also  trigger this check.

Reply:
this is correct - sorry for jumping the gun on this

Reply:
@speth 
If cantera built without "python_full" or "python_minimal" should then "test" trigger the check_for_pytest? Is pytest required just for C++ library and Fortran wrapper testing?

Reply:
No, of course not. But `test` causes the Python test suite to be executed when the Python module is installed, so this check does need to happen in that case.

It's not clear from this line alone, but if you look further down where this test is actually handled, you'll see that it happens in a block that checks that `env['python_package'] != 'none'`.